tests/test_autochunk/test_transformer/test_autochunk_gpt.py

```diff
@@ -0,0 +1,65 @@
+from functools import partial
+from typing import List, Tuple
+
+import pytest
+import torch
+import torch.multiprocessing as mp
+
+try:
+    from transformers import GPT2Config, GPT2Model
```

Comment:
We have prepared many prebuilt test models, you do not need to define them by you self.
```
test_models = ['gpt2', 'bert', 'simple_net', 'repeated_computed_layers', 'nested_model', 'albert']

    for model_name in test_models:
        get_components_func = non_distributed_component_funcs.get_callable(model_name)
        model_builder, train_dataloader, _, _, criterion = get_components_func()
 ```